frontend/src/pages/Profile.tsx

```diff
@@ -135,7 +124,7 @@ const XButton = styled.div`
 export function Profile() {
   const [popup, setPopup] = useState<boolean>(false);
   const [allReferringStaff, setAllReferringStaff] = useState<User[]>([]);
-  const [allHousingLocators, setAllHousingLocators] = useState<User[]>([]);
+  const [, setAllHousingLocators] = useState<User[]>([]);
```

Comment:
```suggestion

```

Since we aren't even using the housing locator state, we can just completely remove this line and remove the 

```javascript
setAllHousingLocators(dataContext.allHousingLocators);
```

line in the `useEffect` lower in the file.